File: src/main/java/net/igneo/icv/entity/weapon/ember/EmberRenderer.java
package net.igneo.icv.entity.weapon.ember;

import net.minecraft.client.renderer.entity.EntityRendererProvider;
import software.bernie.geckolib.renderer.GeoEntityRenderer;
import software.bernie.geckolib.renderer.layer.AutoGlowingGeoLayer;

public class EmberRenderer extends GeoEntityRenderer<EmberEntity> {
    public EmberRenderer(EntityRendererProvider.Context renderManager) {
        super(renderManager, new EmberModel());
        addRenderLayer(new AutoGlowingGeoLayer<>(this));
    }
}

File: src/main/java/net/igneo/icv/entity/weapon/ember/EmberRenderer.kt
package net.igneo.icv.entity.weapon.ember

import net.minecraft.client.renderer.entity.EntityRendererProvider
import software.bernie.geckolib.renderer.GeoEntityRenderer
import software.bernie.geckolib.renderer.layer.AutoGlowingGeoLayer

class EmberRenderer(renderManager: EntityRendererProvider.Context) :
    GeoEntityRenderer<EmberEntity?>(renderManager, EmberModel()) {
    init {
        addRenderLayer(AutoGlowingGeoLayer(this))
    }
}
